dojo/settings/settings.dist.py

```diff
@@ -848,7 +850,7 @@ def saml2_attrib_map_format(dict):
                 'name_id_format': saml2.saml.NAMEID_FORMAT_TRANSIENT,
                 'want_response_signed': False,
                 'want_assertions_signed': True,
-                'force_authn': True,
+                'force_authn': env('DD_SAML2_FORCE_AUTH'),
```

Comment:
Do not use env variables directly, config should support config py files as well 

Please take a look:

https://github.com/italvi/django-DefectDojo/blob/patch-1/dojo/settings/settings.dist.py#L826

Reply:
Actually this is my mistake :/ can you fix this as well?